squash-the-creeps/rust/cheatseet.txt

```diff
@@ -0,0 +1,98 @@
+There are some guidelines you can rely on (which you probably already know):
+For Godot-native methods
+There's no dynamic typing, so you need to cast your object to your desired type before attempting to call methods of that type on it.
+
+Const methods
+Calling can be done with either <&MyClass>::base().method() or <&Gd<MyClass>>::method().
+Both the godot documentation and the gdext-rust bindings specify which are const and which aren't, the gdext bindings specify that through method signatures (eg: fn get_position(&Node2D) vs fn add_child(&mut Node)).
```

Comment:
not `const` (which has its own specific meaning!) but with mutable/immutable receivers.

Reply:
When in doubt consult the docs: https://doc.rust-lang.org/std/keyword.self.html and `The Rust Reference` https://doc.rust-lang.org/reference/expressions/operator-expr.html#borrow-operators